File: petbuddies-ai/src/main/java/br/com/fiap/petbuddies/dto/client/ResponsavelDto.java
package br.com.fiap.petbuddies.dto.client;

public class ResponsavelDto {

    private Long id;
    private String nome;
    private String telefone;
    private String status;

    public ResponsavelDto() {}

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }

    public String getNome() { return nome; }
    public void setNome(String nome) { this.nome = nome; }

    public String getTelefone() { return telefone; }
    public void setTelefone(String telefone) { this.telefone = telefone; }

    public String getStatus() { return status; }
    public void setStatus(String status) { this.status = status; }
}

File: PetBuddies-API/PetBuddies-API/Dtos/Responsavel/ResponsavelDto.cs
namespace PetBuddies_API.Dtos.Responsavel
{
    public class ResponsavelDto
    {
        public int Id { get; init; }

        public string Nome { get; init; } = string.Empty;

        public string Telefone { get; init; } = string.Empty;

        public string Status { get; init; } = string.Empty;
    }
}
